opba-banking-rest-api/src/main/java/de/adorsys/opba/tppbankingapi/mapper/FacadeToRestMapper.java

```diff
@@ -0,0 +1,5 @@
+package de.adorsys.opba.tppbankingapi.mapper;
+
+public interface FacadeToRestMapper<R, F> {
```

Comment:
Probable FacadeResponseBodyToRestBodyMapper is more understandable name

Reply:
ok